src/components/BarChartCard/BarChartCard.jsx

```diff
@@ -41,6 +41,8 @@ const BarChartCard = ({
   isLoading,
   interval,
   className,
+  domainRange,
+  endDate,
```

Comment:
is endDate actually being used anymore?

Reply:
Nope, thought I caught all of those. Removing it now